### PR TITLE
fix '3.1.0' was not found

### PR DESCRIPTION
**Pull request by clanto007**
I tried to fix the problem updating the used SDK to 5.0

### CODE REVIEW

**Comment by captainhook** [comment]
Looks good to me; I will add a few others to reviews before it's merged.

Thank you very much.

**Review by anhyzer5525 — COMMENTED** [review]
After making all of these changes to my instance and then running the update I was successfully able to upgrade and add new passwords into my vault.

Good job on fixing this!

File: src/licenseGen/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1 as build
+FROM mcr.microsoft.com/dotnet/sdk as build
```
Comment (h44z) [inline src/licenseGen/Dockerfile:1]: can we fix the version to :5.0 if we are using netcoreapp5.0 below?

```suggestion
FROM mcr.microsoft.com/dotnet/sdk as build:5.0
```

File: build.sh [hunk 2]
```diff
@@ -13,7 +13,7 @@ echo "Building BitBetter for BitWarden version $BW_VERSION"
 
 cp "$DIR/.keys/cert.cert" "$DIR/src/bitBetter/.keys"
 
-docker run --rm -v "$DIR/src/bitBetter:/bitBetter" -w=/bitBetter mcr.microsoft.com/dotnet/core/sdk:3.1 sh build.sh
+docker run --rm -v "$DIR/src/bitBetter:/bitBetter" -w=/bitBetter mcr.microsoft.com/dotnet/sdk sh build.sh
```
Comment (h44z) [inline build.sh:16]: can we fix the version to :5.0 if we are using netcoreapp5.0 in the other files?

```suggestion
docker run --rm -v "$DIR/src/bitBetter:/bitBetter" -w=/bitBetter mcr.microsoft.com/dotnet/sdk:5.0 sh build.sh

**Review by h44z — CHANGES_REQUESTED** [review]
I think it would be better to fix the new dotnet SDK version to 5.0 as we are referencing this explicit version in some files.

**Comment by clanto007** [comment]
@h44z thank you

File: src/licenseGen/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1 as build
+FROM mcr.microsoft.com/dotnet/sdk as build:5.0
```
Comment (h44z) [inline src/licenseGen/Dockerfile:1]:
```suggestion
FROM mcr.microsoft.com/dotnet/sdk:5.0 as build
```

**Review by h44z — APPROVED** [review]

**Pull request merged**